src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,14 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
-);
+export const MovieList = ({ movies }) => {
+  const moviesCards = movies.map(
+    movie => <MovieCard movie={movie} key={movie.imdbId} />,
+  );
```

Comment:
i think you don't need to create variable in this case

Reply:
Good point. In general we don't need to do this